### ISSUE TITLE
Earth relief files and corruption during download

### ISSUE THREAD

**Issue by ejhgeo** [GenericMappingTools/gmt#3004]
Hello, 
I am having the same problem that was described in issue#549 (Earth relief files and corruption during download) from a while back. I made the mistake of killing the grdcut command:
```
gmt grdcut earth_relief_15s.grd -R-125/-113/31/43 -GHW1/calnv_topo.nc -V
```

Then when I try to rerun the the command it gives the error: 
```
grdcut: Processing input grid
ERROR 4: `/Users/ehightower/.gmt/earth_relief_15s.grd' not recognized as a supported file format.
grdcut: gmt_M_err_trap: 2
grdcut (gmt_api.c:4631(api_import_grid)): Not a supported grid format [earth_relief_15s.grd]
[Session gmt (0)]: Error returned from GMT API: GMT_GRID_READ_ERROR (18)
[Session gmt (0)]: Error returned from GMT API: GMT_GRID_READ_ERROR (18)
[Session gmt (0)]: Error returned from GMT API: GMT_GRID_READ_ERROR (18)
```
Is there any way for the user to resolve this problem at this point? i.e. successfully redownload the built in grids? 

I'm working on OSx 10.15.2 and GMT version 5.4.4

Thanks.

**Comment by PaulWessel** [comment]
You should update to 6.  5.4 workaround would be to go to ~/.gmt and look for that corrupted file and delete it.  You can the download it again.

**Comment by seisman** [comment]
Closing as it was fixed in GMT 6.0.0.
